### DISCUSSION TITLE
M.I.B. 3.6.x US to EU conversion issue?

### DISCUSSION THREAD

**Discussion by dmlion** [Mr-MIBonk/M.I.B._More-Incredible-Bash#247] [General]
Yesterday I've made an EU conversion for my Cayman 718
Originally it had MHI2_US_POG11_P4173 firmware
During successful conversion process (Convert unit to EU - only one option) I saw info about summer time algo and EU available languages dataset
But after installing EU firmware (MHI2_ER_POG11_K5126) I can see only Enlish and Espanol languages. Also time for GMT+3 shows me one hour more (23:00 instead of 22:00)
What can I do to check what went wrong and fix it?

**Comment by harman-f** [comment]
Please share your Log files

**Comment by dmlion** [comment]
[MU1120-JTB-07426.01.18119201C6.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11719090/MU1120-JTB-07426.01.18119201C6.log)
